libmscore/element.cpp

```diff
@@ -603,7 +603,7 @@ void Element::writeProperties(XmlWriter& xml) const
                   xml.setLidLocalIndex(_links->lid(), xml.assignLocalIndex(loc));
                   }
             else {
-                  if (s->links()) {
+                  if (s && s->links()) {
```

Comment:
This part is not /yet) in master, hence 4.x crashes on save, after having added sich a note-achnored line (it is in a different file there though)